wiki/Tournaments/UKCC/6/en.md

```diff
@@ -7,14 +7,796 @@ tags:
+| Streamer | ![][flag_GB] [Electovoid](https://osu.ppy.sh/users/6648263), ![][flag_GB] [helix](https://osu.ppy.sh/users/3322597), ![][flag_GB] [Chrigarr](https://osu.ppy.sh/users/10111130), ![][flag_GB] [\_Takara](https://osu.ppy.sh/users/11599184), ![][flag_CA] [kymotsujason](https://osu.ppy.sh/users/2541804) |
+| Commentator | ![][flag_GB] [hlo](https://osu.ppy.sh/users/2232928), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), ![][flag_GB] [\_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_GB] [Pelzio](https://osu.ppy.sh/users/4775226), ![][flag_GB] [Electovoid](https://osu.ppy.sh/users/6648263), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_GB] [Chrigarr](https://osu.ppy.sh/users/10111130), ![][flag_GB] [sut\_son](https://osu.ppy.sh/users/sut_son), ![][flag_GB] [Nega](https://osu.ppy.sh/users/3181083), ![][flag_FI] [Nyanaro](https://osu.ppy.sh/users/4157611), ![][flag_GB] [Rojaseq](https://osu.ppy.sh/users/3027841), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811) |
+| Designer | ![][flag_GB] [Doomsday is Bad](https://osu.ppy.sh/users/3481378) |
+| Referee | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_GB] [Chrigarr](https://osu.ppy.sh/users/10111130), ![][flag_GB] [\_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [minorman4](https://osu.ppy.sh/users/4837257), ![][flag_GB] [Domblade](https://osu.ppy.sh/users/6701945), ![][flag_FR] [FayeurS 5](https://osu.ppy.sh/users/6177263), ![][flag_GB] [Arthorian](https://osu.ppy.sh/users/10588677), ![][flag_NL] [Wiqued](https://osu.ppy.sh/users/10039350), ![][flag_PL] [Baziu](https://osu.ppy.sh/users/7192659), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_GR] [Nikolomara](https://osu.ppy.sh/users/10077264), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_DE] [real cute](https://osu.ppy.sh/users/9172811), ![][flag_BE] [DraQeu](https://osu.ppy.sh/users/4425520), ![][flag_FR] [\[ Mimir \]](https://osu.ppy.sh/users/7382734) |
+| Statistician | ![][flag_NO] [YokesPai](https://osu.ppy.sh/users/6399568) |
+
 ## Links
 
 - [Discussion thread](https://osu.ppy.sh/community/forums/topics/854898?n=1)
 - [Challonge](https://challonge.com/ukcc6)
 - [Discord](https://discord.gg/EtsU7Cw)
 - [Livestream](https://twitch.tv/osuukcc)
-- [Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vTi6b__EjonaaZCncvq0_zGNjlCRKbfdGXsuZBhLHnhmn4DC2SvRUWlQiqQQQ1vLR6noRyeYr6wAg-y/pubhtml?gid=0#)
+- [Spreadsheet](https://docs.google.com/spreadsheets/d/1AzNYH0Gj2T_ciTde3bSQYISNZJDQ6fDOvTPoamBYf-w/edit#gid=0)
+
+## Participants
+
+### High Seed
```

Comment:
sentence case for each of these headers

Reply:
[Resolved with ](https://github.com/ppy/osu-wiki/pull/6341/commits/682662be8ff6091e687d573febf751a9eece6010)